source/blog/2016-12-14-size-and-change.html.md

```diff
@@ -0,0 +1,165 @@
+group of 5000 (and coincidentally Plato's theoretical optimal size of a
+democracy, 5040), is that it is the size of a group where an individual
+feels that they have no say in the affairs of the group. It is also the
+size at which town meetings in Massachusetts can move from "open to all"
+to representative meetings, open only to elected representatives (6000
+residents). And it is also a size at which companies tend to hit scaling
+challenges both in terms of revenue, cost of innovation, and general
+employee satisfaction (moving from "feeling like part of a family" to
+"feeling like a cog in a machine"). Geoff West has described ([article](http://www.nytimes.com/2010/12/19/magazine/19Urban_West-t.html))  how
+companies ([TED video](https://www.ted.com/talks/geoffrey_west_the_surprising_math_of_cities_and_corporations)) act like organisms&mdash;as they get bigger, their
+growth slows, and become dominated not by innovation, but by economies
+of scale. There are a multitude of articles describing the periods of
+growth, with periodical moments of crisis, in the growth of articles ([here's
+one](https://hbr.org/1983/05/the-five-stages-of-small-business-growth) and [one more](https://hbr.org/1998/05/evolution-and-revolution-as-organizations-grow)). I contend that open source projects that stay centrally
+organized act more like companies, and those who achieve
+hyperdecentralization (what Ori Brafman describes as a "starfish"
+organization in [*The Starfish and the Spider*](https://www.amazon.com/The-Starfish-Spider-Unstoppable-Organizations/dp/1591841836)) act more like ecosystems like forests and
+cities.
+
+## Conclusion
+
+What does this all mean? It means that communities evolve and mutate as
+they grow. The minimum viable infrastructure for a small three-developer
+project is not the same as for a huge ecosystem like OpenStack. There
+will be moments of growth punctuated by moments of unrest&mdash;and at those
+moments, change is needed to allow growth to continue, or the community
+will stagnate and die. Those changes will occur around the boundaries of
+Dunbar's numbers. With each change, something of what went before will
+be lost, causing nostalgia, anxiety, and some discontent that things
+were better before. Good communities will pay attention to these
+emotional consequences too. Perpetuating the founding values of a
+community as it scales is a challenge, and as a community grows a mix of
+dogma, lore, and stories can be used to pass on values.It means that as
+communities grow, group identity, and the sub-groups that grow from
+group identity, needs to be managed&mdash;to avoid the [anti-patterns](http://communitymgt.wikia.com/wiki/Category:Anti-patterns)
+of the [clique](http://communitymgt.wikia.com/wiki/Nepotism), [the water cooler](http://communitymgt.wikia.com/wiki/Water_cooler), or corporate [command and control](http://communitymgt.wikia.com/wiki/Command_and_Control).
```

Comment:
Does this end a bit abruptly? I can't think of a nice way to tie it all together.